pkg/tarsum/tarsum_test.go

```diff
@@ -49,13 +49,13 @@ var testLayers = []testLayer{
 		filename: "testdata/xattr/layer.tar",
 		jsonfile: "testdata/xattr/json",
 		version:  Version0,
-		tarsum:   "tarsum+sha256:e86f81a4d552f13039b1396ed03ca968ea9717581f9577ef1876ea6ff9b38c98"},
+		tarsum:   "tarsum+sha256:ad39b3cb1cc73565c8afb5add3e8126383dff55c6dda6f8e67665ca81f4fb839"},
 	{
 		// Tests next version of TarSum when xattrs are present
 		filename: "testdata/xattr/layer.tar",
 		jsonfile: "testdata/xattr/json",
 		version:  VersionDev,
-		tarsum:   "tarsum.dev+sha256:6235cd3a2afb7501bac541772a3d61a3634e95bc90bb39a4676e2cb98d08390d"},
+		tarsum:   "tarsum.dev+sha256:4eb104c8272535d05c06dd3d8e48aceb4eae0b9206c8151410cfd20839238ac8"},
```

Comment:
If the new fields were not part of the image, updating those tarsums wouldn't be necessary.